test/server/cards/01_SOR/units/SeasonedShoretrooper.spec.js

```diff
@@ -0,0 +1,50 @@
+describe('Seasoned Shoretrooper', function () {
+    integration(function () {
+        describe('Seasoned Shoretrooper\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['wampa', 'battlefield-marine'],
+                        groundArena: ['seasoned-shoretrooper'],
+                        resources: 5
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors']
+                    }
+                });
+            });
+
+            it('should buff him with 6 or more resources', function () {
+                // with 5 resources it should not be buff
+                expect(this.seasonedShoretrooper.getPower()).toBe(2);
+
+                // valball : pass to refresh state as I cheat a bit with resources
+                this.player1.moveCard(this.wampa, 'resource');
+                this.player1.pass();
+
+                // with 6 resources, power should be buff
+                expect(this.player1.resources.length).toBe(6);
+                expect(this.seasonedShoretrooper.getPower()).toBe(4);
+                expect(this.seasonedShoretrooper.getHp()).toBe(3);
+
+                // with 6 or more resources, power should be buff
+                this.player1.moveCard(this.battlefieldMarine, 'resource');
```

Comment:
What's the reason for going up to 7 resources? Feels like just testing at 6 should be fine and this is adding complexity to the test

Reply:
checking I didn't fail my condition with an equal and not an superior and equal ?